Fixes the error:
In file included from ../nptl/sysdeps/unix/sysv/linux/libc-lowlevellock.c:21:
../nptl/sysdeps/unix/sysv/linux/lowlevellock.c: In function '__lll_lock_wait_private':
../nptl/sysdeps/unix/sysv/linux/lowlevellock.c:34: warning: implicit declaration of function 'THREAD_GETMEM'
../nptl/sysdeps/unix/sysv/linux/lowlevellock.c:34: error: 'THREAD_SELF' undeclared (first use in this function)
../nptl/sysdeps/unix/sysv/linux/lowlevellock.c:34: error: (Each undeclared identifier is reported only once
../nptl/sysdeps/unix/sysv/linux/lowlevellock.c:34: error: for each function it appears in.)
../nptl/sysdeps/unix/sysv/linux/lowlevellock.c:34: error: 'header' undeclared (first use in this function)

Copied from http://www.nabble.com/arm-linux-compilation-failure-and-possible-fix-td19229074.html

--- ./ports/sysdeps/unix/sysv/linux/arm/nptl/lowlevellock.h.orig	2008-10-04 20:46:13.000000000 +0200
+++ ./ports/sysdeps/unix/sysv/linux/arm/nptl/lowlevellock.h	2008-10-04 20:45:40.000000000 +0200
@@ -25,6 +25,7 @@
 #include <atomic.h>
 #include <sysdep.h>
 #include <kernel-features.h>
+#include <tls.h>
 
 #define FUTEX_WAIT		0
 #define FUTEX_WAKE		1
